lib/techs/tmpl-spec.js

```diff
@@ -69,10 +68,9 @@ module.exports = require('enb/lib/build-flow').create()
         }
         its = Object.keys(its);
 
-        return readAssets.spread(function (asset, it, iti18n) {
```

Comment:
After this change results contain templates with mock of `BEM.I18N` if `langs: false`. Mock of `BEM.I18N` should be so only if the value of `langs` is `true`.